.github/workflows/docker-lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: Docker Linting
+
+on:
+  push:
```

Comment:
Can we trigger this on pull requests instead of push event?

Reply:
Yes, it should run on pull-request. And if we are going to run it on pull-request, then the issues identified by the tool should also be fixed in this PR. There are some issue i can see in the screenshot shared in the ticket identified by the tool.